src/colorizer/AnnotationData.ts

```diff
@@ -276,6 +278,14 @@ export class AnnotationData implements IAnnotationData {
     this.timeToLabelIdMap = null;
   }
 
+  setLabelOnIds(labelIdx: number, ids: number[], value: boolean): void {
```

Comment:
Do you no longer need a second function `setLabelOnId` since you can just pass single or multiple ids as arrays?

Reply:
I guess not, I'll see how many places I'd need to update if I change it.